microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+*  Additional options used to create WebView2 Environment to manage custom scheme registration.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2EnvironmentOptions Class](#)
+   * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](#)
+* [CoreWebView2CustomSchemeRegistration Class](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2EnvironmentOptions Class](#)
+* [CoreWebView2CustomSchemeRegistration Class](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2EnvironmentOptions4 interface](#)
+   * [ICoreWebView2EnvironmentOptions4::GetCustomSchemeRegistrations method](#)
+   * [ICoreWebView2EnvironmentOptions4::SetCustomSchemeRegistrations method](#)
+* [ICoreWebView2CustomSchemeRegistration interface](#)
+   * [ICoreWebView2CustomSchemeRegistration::GetAllowedOrigins method](#)
+   * [ICoreWebView2CustomSchemeRegistration::SetAllowedOrigins method](#)
+   * [ICoreWebView2CustomSchemeRegistration::HasAuthorityComponent property (get](#), [put)](#)
+   * [ICoreWebView2CustomSchemeRegistration::SchemeName property (get)](#)<!--no put-->
+   * [ICoreWebView2CustomSchemeRegistration::TreatAsSecure property (get](#), [put)](#)
+
+---
+
+<!-- ====================================================================== -->
+
+## 1.0.1671-prerelease
+
+Release Date: Feb 13, 2023  
+
+[NuGet package for WebView2 SDK 1.0.1671](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1671-prerelease)  
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 112.0.1671.0 or higher. 
+
+### General
+
+#### Experimental features 
+
+*  Added support for the Experimental File API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+* [CoreWebView2ExperimentalFile interface](#)
+   * [CoreWebView2ExperimentalFile::get_Path method](#)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+* [CoreWebView2ExperimentalFile interface](#)
+   * [CoreWebView2ExperimentalFile::get_Path method](#)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalFile interface](#)
+   * [ICoreWebView2ExperimentalFile::get_Path method](#)
+* [ICoreWebView2ExperimentalWebMessageReceivedEventArgs interface](#)
+
+---
+
+*  Added support for Experimental Object Collection View API
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [CoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [CoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)

```

Comment:
tech review this change.  For WinRT, should the tab just contain "n/a"?
```suggestion
* [CoreWebView2ExperimentalObjectCollectionView Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2experimentalobjectcollectionview?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)
   * [CoreWebView2ExperimentalObjectCollectionView.Count Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2experimentalobjectcollectionview?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#count)
   * [CoreWebView2ExperimentalObjectCollectionView.GetValueAtIndex Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2experimentalobjectcollectionview?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#getvalueatindex)
```

Reply:
@peiche-jessica - Does CoreWebView2ExperimentalObjectCollectionView class exist for WinRT?  If not, I'll put "n/a" in this tab.

Reply:
I might delete this comment thread and add it to the Suggested Change comment instead.